### PR TITLE
Allow computing number of cell vertices in `compute_cell_sizes`

### PR DESCRIPTION
**Pull request by MatthewFlamm**
### Overview

Add option to include 'VertexCount' in the returned data.

### Details

This is needed for implementing https://github.com/MatthewFlamm/pyransame/issues/29.

I opted to default to `False` for backwards compatibility. It should also be a rare usage.

### CODE REVIEW

File: pyvista/core/filters/data_set.py [hunk 1]
```diff
@@ -2068,12 +2070,16 @@ def compute_cell_sizes(self, length=True, area=True, volume=True, progress_bar=F
         progress_bar : bool, default: False
             Display a progress bar to indicate progress.
 
+        vertex_count : bool, default: False
+            Specify whether or not to computer the number of vertices
```
Comment (MatthewFlamm) [inline pyvista/core/filters/data_set.py:2074]:
```suggestion
            Specify whether or not to compute the number of vertices
```

File: pyvista/core/filters/data_set.py [hunk 2]
```diff
@@ -2068,12 +2070,16 @@ def compute_cell_sizes(self, length=True, area=True, volume=True, progress_bar=F
...
+            in 0D cells.
```
Comment (MatthewFlamm) [inline pyvista/core/filters/data_set.py:2075]:
```suggestion
            of 0D cells.
```

File: pyvista/core/filters/data_set.py [hunk 3]
```diff
@@ -2068,12 +2070,16 @@ def compute_cell_sizes(self, length=True, area=True, volume=True, progress_bar=F
         progress_bar : bool, default: False
             Display a progress bar to indicate progress.
 
+        vertex_count : bool, default: False
+            Specify whether or not to compute the number of vertices
+            of 0D cells.
```
Comment (tkoyama010) [inline pyvista/core/filters/data_set.py:2075]: I rewrite the docstring based on the [SetComputeVertexCount](https://vtk.org/doc/nightly/html/classvtkCellSizeFilter.html#af459249cf230a59304e86fbaf9503099) manual. If my understanding is correct, please approve.
```suggestion
            Specify whether or not to compute sizes for vertices and 0D cells.
            The computed value is the number of points in the cell.
```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]
Thanks for the change. I'm glad to see that `pyransame` development is going well. I have a suggestion about the parameter description.

File: pyvista/core/filters/data_set.py [hunk 4]
```diff
@@ -2068,12 +2070,16 @@ def compute_cell_sizes(self, length=True, area=True, volume=True, progress_bar=F
         progress_bar : bool, default: False
             Display a progress bar to indicate progress.
 
+        vertex_count : bool, default: False
+            Specify whether or not to compute sizes for vertices and 0D cells.
```
Comment (MatthewFlamm) [inline pyvista/core/filters/data_set.py:2074]:
```suggestion
            Specify whether or not to compute sizes for vertex and polyvertex cells.
```

**Review by tkoyama010 — APPROVED** [review]
LGTM

**Pull request merged**